src/core/src/core_logic/PatchInstaller.py

```diff
@@ -165,9 +170,6 @@ def install_updates(self, maintenance_window, package_manager, simulate=False):
 
         self.composite_logger.log("\n\nInstalling patches in sequence...")
```

Comment:
Do we need to remove or modify this log?

Reply:
Removed this log as it is not much useful. There are other logs in this execution path which can be used during debug.